justfile

```diff
@@ -0,0 +1,62 @@
+name := 'cosmic-osd'
+
+polkit-agent-helper-1 := /usr/libexec/polkit-agent-helper-1
+
+rootdir := ''
+prefix := '/usr'
+
+base-dir := absolute_path(clean(rootdir / prefix))
+
```

Comment:
Suggestion: Add a line like `cargo-target-dir := env('CARGO_TARGET_DIR', 'target')` here to read the CARGO_TARGET_DIR environment variable.